config/initializers/assets.rb

```diff
@@ -96,7 +96,7 @@
       Foreman::Plugin.all.each do |plugin|
         # Manifests may be stored under the engine installation or under the Foreman app root, then
         # either with just the plugin name or with hyphens replaced with underscores.
-        manifest_path = File.join(plugin.path, "/public/webpack/#{plugin.id}/manifest.json")
+        manifest_path = File.join(plugin.path, 'public', 'webpack', plugin.id, 'manifest.json')
```

Comment:
This needs to be `plugin.id.to_s` or else it throws an error:

```
/home/vagrant/foreman/config/initializers/assets.rb:99:in `join': no implicit conversion of Symbol into String (TypeError)
```

Reply:
Updated. Also removed the comment which is not relevant for this line.